google-cloud-examples/src/main/java/com/google/cloud/examples/securitycenter/snippets/v1p1beta1/NotificationConfigSnippets.java

```diff
@@ -0,0 +1,169 @@
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.examples.securitycenter.snippets.v1p1beta1;
+
+import com.google.cloud.securitycenter.v1p1beta1.CreateNotificationConfigRequest;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfig;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfig.EventType;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfig.StreamingConfig;
+import com.google.cloud.securitycenter.v1p1beta1.NotificationConfigName;
+import com.google.cloud.securitycenter.v1p1beta1.OrganizationName;
+import com.google.cloud.securitycenter.v1p1beta1.SecurityCenterClient;
+import com.google.cloud.securitycenter.v1p1beta1.SecurityCenterClient.ListNotificationConfigsPagedResponse;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.FieldMask;
+import java.io.IOException;
+
+/** Snippets for how to work with Notifications in Cloud SCC. */
+final class NotificationConfigSnippets {
+  private NotificationConfigSnippets() {}
+
+  public static NotificationConfig createNotificationConfig(
+      String organizationId, String pubsubTopic, String notificationConfigId) {
+
+    // [START scc_create_notification_config]
+    // TODO(UpdateMe) String organizationId = "{your-org-id}";
+    // TODO(UpdateMe) String notificationConfigId = "config-id";
+    // TODO(UpdateMe) String pubsubTopic = "projects/{your-project}/topics/{your-topic}";
+    // Ensure this ServiceAccount has the "pubsub.topics.setIamPolicy" permission on the topic.
+
+    String orgName = String.format("organizations/%s", organizationId);
+
+    try (SecurityCenterClient client = SecurityCenterClient.create()) {
+      CreateNotificationConfigRequest request =
+          CreateNotificationConfigRequest.newBuilder()
+              .setParent(orgName)
+              .setConfigId(notificationConfigId)
+              .setNotificationConfig(
+                  NotificationConfig.newBuilder()
+                      .setDescription("Java notification config")
+                      .setPubsubTopic(pubsubTopic)
+                      .setEventType(EventType.FINDING)
+                      .setStreamingConfig(
+                          StreamingConfig.newBuilder().setFilter("state = \"ACTIVE\"").build())
+                      .build())
+              .build();
+
+      NotificationConfig response = client.createNotificationConfig(request);
+      System.out.println(String.format("Notification config was created: %s", response));
+      return response;
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
Not sure this is actually useful to anyone.

Reply:
Just threw the IOException